src/openms/include/OpenMS/CHEMISTRY/EnzymaticDigestion.h

```diff
@@ -122,6 +123,9 @@ namespace OpenMS
     /// @note: the protein sequence must be unmodified (i.e. only amino acid characters, no modifications or masses in brackets)
     bool isValidProduct(const String& protein, Size pep_pos, Size pep_length, bool methionine_cleavage = false, bool ignore_missed_cleavages = true) const;
 
+    ///
+    bool filterByMissingCleavages(const String& sequence, std::function<bool(const Int)> filter) const;
```

Comment:
missing doc. The interface is pretty contrived, so I think its important to have it.

sequence: is an unmodified(!) peptide sequence
filter: ....

Also, the variable name in the definition for sequence is different